test/e2e/reconciler/workspace/controller_test.go

```diff
@@ -141,7 +144,7 @@ func TestWorkspaceController(t *testing.T) {
 					if workspace.Spec.Cluster == "" {
 						return false, fmt.Sprintf("spec.cluster is empty\n%s", toYAML(t, workspace))
 					}
-					if expected := previouslyValidShard.Spec.BaseURL + "/clusters/" + workspace.Spec.Cluster; workspace.Spec.URL != expected {
+					if expected := rootShard.Spec.ExternalURL + "/clusters/" + workspace.Spec.Cluster; workspace.Spec.URL != expected {
```

Comment:
Not quite sure why/how this was passing before. We always set the workspace's URL to $shardExternalURL/clusters/$clusterName